.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,33 @@
 <!--
-Thank you for creating a pull request. Before submitting, please note the following:
+  ⚡️ katchow! We ❤️ Pull Requests!
 
-* If your pull request implements a new feature, please raise an issue to discuss it before sending code. In many cases features are absent for a reason.
-* This message body should clearly illustrate what problems it solves. If there are related issues, remember to reference them.
-* Ideally, include a test that fails without this PR but passes with it. PRs will only be merged once they pass CI. (Remember to `npm run lint`!)
```

Comment:
These guidelines seem important to me to keep. Or perhaps reword into the template?

Reply:
The first point (about an accompanying issue)  I've found that with new features, there's often more work done on implementation before discussion starts, if the contribution is coming from outside contributors, which is then discussed directly on the PR. I've also found it can be a hinderance to contribution if there's too much process _but I'm completely OK with adding that back in_. If you feel there's more value to the process than risk of dissuading contributions, I'm good with that.

Reply:
I can get behind what you're saying certainly in not crowding the process here.

Perhaps a note saying they should research previous discussion on the feature by searching the issues first?

Reply:
Great idea. Definitely a step that should always happen.

Reply:
Committed proposed additions based on that feedback.